README.md

```diff
@@ -41,14 +41,12 @@ cargo install --path helix-term
 
 This will install the `hx` binary to `$HOME/.cargo/bin`.
 
-Now copy the `runtime/` directory somewhere. Helix will by default look for the
-runtime inside the same folder as the executable, but that can be overriden via
-the `HELIX_RUNTIME` environment variable.
-
-> NOTE: You should set this to <path to repository>/runtime in development (if
-> running via cargo).
-> 
-> `export HELIX_RUNTIME=$PWD/runtime`
+Now copy the `runtime/` directory somewhere. Helix will by default look for the runtime
+inside the config directory or the same directory as executable, but that can be overriden
+via the `HELIX_RUNTIME` environment variable.
```

Comment:
Maybe you might want to mention that one can just soft-link the runtime to the git repo runtime to be easier?